### PR TITLE
deleted attrs in netcdf files 

### LINKED ISSUE #214: confusing attrs on the sample data
```python
>>> xr.open_mfdataset(os.path.join(rootdir, '00050101.precip_monthly.nc'), decode_times=False)
<xarray.Dataset>
Dimensions:            (lat: 64, latb: 65, lon: 128, lonb: 129, nv: 2, time: 12)
Coordinates:
  * lonb               (lonb) float64 -1.406 1.406 4.219 7.031 9.844 12.66 ...
  * lon                (lon) float64 0.0 2.812 5.625 8.438 11.25 14.06 16.88 ...
  * latb               (latb) float64 -90.0 -86.58 -83.76 -80.96 -78.16 ...
  * lat                (lat) float64 -87.86 -85.1 -82.31 -79.53 -76.74 ...
  * time               (time) float64 1.476e+03 1.504e+03 1.535e+03 ...
  * nv                 (nv) float64 1.0 2.0
Data variables:
    condensation_rain  (time, lat, lon) float64 5.905e-06 5.728e-06 ...
    convection_rain    (time, lat, lon) float64 0.0 0.0 0.0 0.0 0.0 ...
    time_bounds        (time, nv) float64 1.46e+03 1.491e+03 1.491e+03 ...
    average_DT         (time) float64 31.0 28.0 31.0 30.0 31.0 30.0 31.0 ...
    zsurf              (lat, lon) float64 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 ...
Attributes:
    grid_tile:  N/A
    title:      FMS Model results
    grid_type:  regular
    filename:   00050101.atmos_month.nc
```
The filename attribute confused me because I wasn't sure whether or not `00050101.atmos_month.nc` was created somewhere on my machine.

### PR DESCRIPTION
**Pull request by micahkim23**
Closes #214

### CODE REVIEW

**Review by spencerkclark — APPROVED** [review]
@micahkim23 I downloaded the files and had a look offline; this looks good to me, thanks!  I will let @spencerahill have the final word.

**Comment by spencerahill** [comment]
Thanks for checking @spencerkclark -- I believe you that they're good.

@micahkim23, FYI for a PR like this that is in reference to a specific issue (#214 in this case), it's good practice to write "Closes #<issue number>", a.k.a. in this case "Closes #214", as the first line of the PR description.  That provides context to others who come across it, and also Github then automatically closes the Issue once the PR gets merged.

Also, typically we add a note to the [What's New](https://github.com/spencerahill/aospy/blob/develop/docs/whats-new.rst) section of our docs whenever we merge in something new.  This particular one is small enough that we would normally skip it, but I want your handle on there!  So please add a note under the v0.2.1 section, under a new subheading titled "Enhancements".  Just emulate the other items in that page for the formatting/style.

Thanks!

**Comment by micahkim23** [comment]
@spencerahill I added my changes in the enhancement section of What's New.

File: docs/whats-new.rst
Comment (spencerahill) [inline docs/whats-new.rst]: Strictly speaking, these are attributes of the netCDF files themselves, in addition to the `xr.Dataset` representations of the files.  So replace "``xarray.Dataset``" attributes with "potentially confusing".

**Comment by spencerahill** [comment]
@micahkim23 one last nit, then I'll merge.

**Comment by micahkim23** [comment]
@spencerahill sweet, made the fix.

**Comment by spencerahill** [comment]
In it goes!  Thanks again, and congrats on your first merged PR 🎉 

Normally we would only merge if all of the continuous integration (CI) tests passed.  But the one failure in Travis CI appears to be unrelated, likely a temporary system hangup.  So I think we can safely ignore it.

**Pull request merged**
